apps/calendar/js/app.js

```diff
@@ -129,7 +130,32 @@ function setupControllers() {
   alarms.autoQueue = true;
 }
 
+function setupIntlHelpers() {
```

Comment:
Can we put this in its own module? We're trying in general to reduce the size of our main app file.